### PR TITLE
ELPP-3490 Increase maximum sizes of aggregated terms

### PR DESCRIPTION
**Pull request by giorgiosironi**

### CODE REVIEW

File: src/Search/Api/Elasticsearch/ElasticQueryBuilder.php [hunk 1]
```diff
@@ -16,14 +16,17 @@
     const DATE_DEFAULT = 'sortDate';
     const DATE_PUBLISHED = 'published';
 
+    const MAXIMUM_TYPES = 100;
```
Comment (thewilkybarkid) [inline src/Search/Api/Elasticsearch/ElasticQueryBuilder.php]: `MAXIMUM_SUBJECTS`?

File: src/Search/Api/Elasticsearch/ElasticQueryBuilder.php [hunk 2]
```diff
@@ -16,14 +16,17 @@
...
+
     private $dateType;
 
     public function __construct(string $index)
     {
         $this->query['index'] = $index;
         $this->query['body']['aggregations']['type_agg']['terms'] = [
             'field' => '_type',
-            'size' => 18,
+            'size' => self::MAXIMUM_TYPES,
```
Comment (thewilkybarkid) [inline src/Search/Api/Elasticsearch/ElasticQueryBuilder.php]: This is a constant number, so ok to leave as 18.

Comment (thewilkybarkid) [reply src/Search/Api/Elasticsearch/ElasticQueryBuilder.php]: Though I count it as 17!

Comment (giorgiosironi) [reply src/Search/Api/Elasticsearch/ElasticQueryBuilder.php]: I'll leave the constant and put it to 18

Comment (giorgiosironi) [reply src/Search/Api/Elasticsearch/ElasticQueryBuilder.php]: `curl -v https://prod--gateway.elifesciences.org/search | jq '.types | length'` is 17 in fact.

Comment (giorgiosironi) [reply src/Search/Api/Elasticsearch/ElasticQueryBuilder.php]: It seems like the kind of things that will break with no one noticing

**Comment by giorgiosironi** [comment]
Let's see what this does.

**Pull request merged**